src/components/01-branding/02-typography/fields.njk

```diff
@@ -1,9 +1,71 @@
 <h1>Fields</h1>
-<p>Examples of how to display fields.</p>
 
-<h2>Field</h2>
-<div>Display general field.</div>
-<sds-page class="condensed">
+<h2>Inline Field</h2>
+<p>Display name and value inline.</p>
+<div class="grid-row">
+    <div class="sds-field">
+        <div class="sds-field__name">Field:</div>
+        <div class="sds-field__value">Lorem ipsum dolor</div>
+    </div>
+</div>
+<div class="grid-row">
+    <div class="sds-field sds-field--featured">
+        <div class="sds-field__name">Featured Field:</div>
+        <div class="sds-field__value">Lorem ipsum dolor</div>
+    </div>
+</div>
+<div class="grid-row">
+    <div class="sds-field sds-field--small">
+        <div class="sds-field__name">Field Small:</div>
+        <div class="sds-field__value">Lorem ipsum dolor</div>
+    </div>
+</div>
+<div class="grid-row">
+    <div class="sds-field sds-field--large">
+        <div class="sds-field__name">Field Large:</div>
+        <div class="sds-field__value">Lorem ipsum dolor</div>
+    </div>
+</div>
+<div class="grid-row">
+    <div class="sds-field sds-field--parragraph">
+        <div class="sds-field__name">Parragraph Field:</div>
+        <div class="sds-field__value">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque vehicula, mi at imperdiet imperdiet, lectus velit vehicula dolor, ac scelerisque metus lectus in eros. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos.</div>
+    </div>
+</div>
+<h2>Stacked Field</h2>
+<p>Display name and value stacked.</p>
+<div class="grid-row">
+    <div class="sds-field sds-field__stacked">
```

Comment:
change to sds-field--stacked (everywhere it was changed)

Reply:
fixed